app/dashboard/templates/shared/rating.html

```diff
@@ -15,16 +15,24 @@
   along with this program. If not, see <http://www.gnu.org/licenses/>.
 {% endcomment %}
 {% load i18n %}
-<div class="w-100 mt-2" style="height: 70px;">
-<label class="form__label" for="notificationEmail">{% trans "Rate your experience with the " %}{{ counterparty }}</label>
-  <div class="rating">
+<div class="w-100 p-2">
+<!-- <label class="form__label" for="notificationEmail">{% trans "Rate your experience with the " %}{{ counterparty }}</label> -->
```

Comment:
All unused commented code should be removed

Reply:
yes thanks, is a WIP for now just some static front end.

Reply:
OK :)